db/Guardfile

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
```

Comment:
Why is this file popping up in all of your PRs? :))

I really don't expect changes to it to happen so often. Please don't forget to do self code reviews.

Reply:
ok

Reply:
![image](https://user-images.githubusercontent.com/24190790/130761318-7d2e635f-fe83-4251-ade6-15036a30b9bb.png)